docs/alps/hardware.md

```diff
@@ -40,20 +40,41 @@ Alps was installed in phases, starting with the installation of 1024 AMD Rome du
 
 There are currently four node types in Alps, with another becoming available in 2025:
 
-| type           | blades | nodes | CPU sockets | GPU devices |
-| ----           | ------:| -----:| -----------:| -----------:|
-| NVIDIA GH200   | 1344   | 2688  | 10,752      | 10,752      |
-| AMD Rome       |  256   | 1024  |  2,048      | --          |
-| NVIDIA A100    |   72   |  144  |    144      | 576         |
-| AMD MI250x     |   12   |   24  |     24      |  96         |
-| AMD MI300A     |   64   |  128  |    512      | 512         |
+| type           | abbreviation  | blades | nodes | CPU sockets | GPU devices |
+| ----           | -------       | ------:| -----:| -----------:| -----------:|
+| NVIDIA GH200   | gh200         | 1344   | 2688  | 10,752      | 10,752      |
+| AMD Rome       | zen2          |  256   | 1024  |  2,048      | --          |
+| NVIDIA A100    | a100          |   72   |  144  |    144      | 576         |
+| AMD MI250x     | mi200         |   12   |   24  |     24      |  96         |
+| AMD MI300A     | mi300         |   64   |  128  |    512      | 512         |
 
 [](){#ref-alps-gh200-node}
 ### NVIDIA GH200 GPU Nodes
 
-!!! todo
+!!! under-construction
+    The description of the GH200 nodes is incomplete.
+
+    Let us know if there is missing information.
```

Comment:
I feel like this is redundant, we already know that there is missing information (and say so in the sentence before).

I guess you're going for something like "Let us know if there's specific information you'd like to see documented first"? I don't like my wording there, but is that what you had in mind with the sentence?

Reply:
![image](https://github.com/user-attachments/assets/93a8b3f2-67e0-47d6-b9dd-1b9adba9af2d)

Reply:
That looks great, thanks @bcumming!